modules/fts/pages/fts-geospatial-queries.adoc

```diff
@@ -6,21 +6,32 @@ Geospatial queries return documents that contain location-data.
 
-A _geospatial query_ specifies a location, and returns each document that contains a proximity-match.
-A location is represented by means of _longitude_-_latitude_ coordinate pairs.
+A _geospatial query_ specifies an area, and returns each document that contains a reference to a location within the area.
+Areas and locations are represented by means of _latitude_-_longitude_ coordinate pairs.
 
-This allows an application, based on the user's input of a particular coordinate, address, or property name, to derive a list of _points of interest_ that lie within a specified distance of the user-referenced location; and display these for the user's benefit.
+This allows an application, based on the user's input of a particular coordinate, address, or property name, to derive a list of _points of interest_ that lie within an area that surrounds the user-referenced location; and display these for the user's benefit.
 
-The location-data provided by a geospatial query can be either of the following:
+A geospatial query can specify an area by means of:
 
-* A location, specified as a longitude-latitude coordinate pair; and a distance, in miles.
+* A location, expressed as a latitude-longitude coordinate pair; and a distance, in miles.
 The location determines the center of a circle whose radius-length is the specified distance.
 Documents are returned if they reference a location within the circle.
-* Two longitude-latitude coordinate pairs.
-These are respectively taken to indicate the top left and bottom right corners of a bounding box.
+
+* Two latitude-longitude coordinate pairs.
+These are respectively taken to indicate the top left and bottom right corners of a _rectangular_ bounding box.
+Documents are returned if they reference a location within the box.
+
+* An array of three of more latitude-longitude coordinate pairs.
+Each of the pairs is taken to indicate one corner of a _polygonal_ bounding box.
 Documents are returned if they reference a location within the box.
 
-To be successful, a geospatial query must reference an index within which the _geopoint_ type mapping has been applied to the field containing the target longitude-latitude coordinate pair.
+To be successful, a geospatial query must reference an index within which the _geopoint_ type mapping has been applied to the field containing the target latitude-longitude coordinate pair.
+
+Geospatial queries return _all_ documents whose locations are within the query-specified area.
+To specify _holes_ within the area, so that one or more subsets of returned documents can be omitted from the final results, boolean queries should be applied to the set of documents returned by the geospatial query.
+See xref:fts:fts-query-types.adoc[Query Types].
+
+For any geospatial query, latitude-longitude coordinate pairs can also be specified as _geohashes_, as demonstrated in xref:fts:fts-geospatial-queries.adoc#specifying-polygon-points[Specifying Polygon Points], below.
```

Comment:
latitude-longitude can be expressed in any of the 4 accepted formats and this is applicable not only to polygon query but all geo queries.
ref - https://blog.couchbase.com/introducing-geo-bounded-polygon-query-in-full-text-search/
You may find all 4 format types there..
This needs to be documented somewhere.

Reply:
I've put in a new section, to cover this.